Rebuild v4.18.0

ELundby45 opened the pull request:
# Rebuild v4.18.0

[huggingface_hub-feedstock](https://github.com/AnacondaRecipes/huggingface_hub-feedstock) was updated and a change was introduced preventing building the existing `transformers-feedstock`. This rebuild pins `huggingface_hub` to the previous version to prevent the following error: 

`ImportError: cannot import name ‘login’ from ‘huggingface_hub.hf_api’`

Offending PR: https://github.com/huggingface/huggingface_hub/pull/1041/files

ELundby45 committed:
Bump build

ELundby45 committed:
Pin huggingface_hub

ELundby45 committed:
Update about section

anaconda-pkg-build commented:
Linter check failed, with the following output:
The following problems have been found:

ERROR: /tmp/abs_9d86ghy2my/clone/recipe/meta.yaml:34: version_constraints_missing_whitespace: Packages and their version constraints must be space separated
Errors were found

ELundby45 committed:
Be more specific on when breaking change was introduced

https://github.com/huggingface/huggingface_hub/blob/v0.9.1/src/huggingface_hub/__init__.py#L177

anaconda-pkg-build commented:
Linter check failed, with the following output:
The following problems have been found:

ERROR: /tmp/abs_518x8972uj/clone/recipe/meta.yaml:34: version_constraints_missing_whitespace: Packages and their version constraints must be space separated
Errors were found

ELundby45 committed:
Remove outdated win32 constraint.

anaconda-pkg-build commented:
Linter check failed, with the following output:
The following problems have been found:

ERROR: /tmp/abs_5aw29rfd8m/clone/recipe/meta.yaml:35: version_constraints_missing_whitespace: Packages and their version constraints must be space separated
Errors were found

psteyer reviewed (approved):
LGTM

varlackc reviewed (commented):
It seems that there is a redirect on the `doc_url`:
doc_url: https://huggingface.co/docs/transformers/index

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_91jxjazu61/clone/recipe/meta.yaml:35: version_constraints_missing_whitespace: Packages and their version constraints must be space separated
Errors were found

</details>
